app/views/teacher_mailer/application_received.text.erb

```diff
@@ -11,11 +11,16 @@ We are waiting to receive the written evidence you’ve requested from your teac
 # What happens next
 
 <% if @application_form.teaching_authority_provides_written_statement %>
-Once the written evidence is received and checked, your application will be entered into a queue and assigned a QTS assessor. This can take several weeks.
+We are waiting to receive the written evidence you’ve requested from your teaching authority.
+
+Once the written evidence is received and checked, your application will be assessed by a trained assessor. They will check all the information you have submitted. Some information may need verification by third parties.
+
+If we need more information, we will email you. You do not need to contact us.
```

Comment:
This line appears in both conditions of the `if`, so perhaps makes sense to move it outside the `if`.